docs/gpu_determinism.rst

```diff
@@ -0,0 +1,15 @@
+Determinism on GPUs
+======================
+
+It can be tricky to make floating point computations on GPUs fully
+deterministic, in the sense of computing precisely the same values
+across separate program executions on the same hardware, and in general doing
+so may require sacrificing some performance. Some sources of nondeterminism in
+XLA:GPU are:
```

Comment:
I'm unsure whether this is actually true of the readthedocs rst renderer, but github's syntax highlighting is parsing `:GPU are:` as a rst directive. Maybe just write `XLA/GPU`?